client/src/rpc_custom_error.rs

```diff
@@ -106,6 +110,11 @@ impl From<RpcCustomError> for Error {
                 message: "No snapshot".to_string(),
                 data: None,
             },
+            RpcCustomError::BigTableSlotSkipped { slot } => Self {
+                code: ErrorCode::ServerError(JSON_RPC_SERVER_ERROR_BIGTABLE_SLOT_SKIPPED),
+                message: format!("Slot {} was skipped, or missing in BigTable", slot),
```

Comment:
@mvines how do you feel about this wording?
I originally made this error more generic so it could work for blocks or signatures, but users seem happy with `null` responses for signatures.

Reply:
Let's not reveal BigTable to the caller, that's in implementation detail.  PersistentLedgerStorage or LongTermStorage perhaps?   ArWeave/PostreSQL are possible other backends that might be plugged in one day